azure-ml-deployment/deploymentconfigAKS.json

```diff
@@ -0,0 +1,21 @@
+{
+    "computeType": "aks",
+    "autoScaler":
+    {
+        "autoscaleEnabled": true,
+        "minReplicas": 1,
+        "maxReplicas": 3,
+        "refreshPeriodInSeconds": 1,
+        "targetUtilization": 70
+    },
+    "dataCollection":
+    {
+        "storageEnabled": true
+    },
+    "authEnabled": true,
+    "containerResourceRequirements":
+    {
+        "cpu": 0.5,
+        "memoryInGB": 1.0
```

Comment:
what is the maximum here? 1 GB? that seems really small considering...

Reply:
This example is indeed a deployment config for K8s on Azure (aka AKS) these files are meant to be modified by the user.